page-objects/guest-page.js

```diff
@@ -0,0 +1,140 @@
+class Guestacc {
```

Comment:
@karthikeyan7010 When I look at 3 classes, all 3 has same methods and behavior. Whole lot of duplicated codes. pls fix it